<!DOCTYPE html>
<html>
<head>
<title>P-tree</title>
</head>
<body>
<div class="mw-parser-output"><style data-mw-deduplicate="TemplateStyles:r1033289096">.mw-parser-output .hatnote{font-style:italic}.mw-parser-output div.hatnote{padding-left:1.6em;margin-bottom:0.5em}.mw-parser-output .hatnote i{font-style:normal}.mw-parser-output .hatnote+link+.hatnote{margin-top:-0.5em}</style>

<style data-mw-deduplicate="TemplateStyles:r1045330069">.mw-parser-output .sidebar{width:22em;float:right;clear:right;margin:0.5em 0 1em 1em;background:#f8f9fa;border:1px solid #aaa;padding:0.2em;text-align:center;line-height:1.4em;font-size:88%;border-collapse:collapse;display:table}body.skin-minerva .mw-parser-output .sidebar{display:table!important;float:right!important;margin:0.5em 0 1em 1em!important}.mw-parser-output .sidebar-subgroup{width:100%;margin:0;border-spacing:0}.mw-parser-output .sidebar-left{float:left;clear:left;margin:0.5em 1em 1em 0}.mw-parser-output .sidebar-none{float:none;clear:both;margin:0.5em 1em 1em 0}.mw-parser-output .sidebar-outer-title{padding:0 0.4em 0.2em;font-size:125%;line-height:1.2em;font-weight:bold}.mw-parser-output .sidebar-top-image{padding:0.4em}.mw-parser-output .sidebar-top-caption,.mw-parser-output .sidebar-pretitle-with-top-image,.mw-parser-output .sidebar-caption{padding:0.2em 0.4em 0;line-height:1.2em}.mw-parser-output .sidebar-pretitle{padding:0.4em 0.4em 0;line-height:1.2em}.mw-parser-output .sidebar-title,.mw-parser-output .sidebar-title-with-pretitle{padding:0.2em 0.8em;font-size:145%;line-height:1.2em}.mw-parser-output .sidebar-title-with-pretitle{padding:0.1em 0.4em}.mw-parser-output .sidebar-image{padding:0.2em 0.4em 0.4em}.mw-parser-output .sidebar-heading{padding:0.1em 0.4em}.mw-parser-output .sidebar-content{padding:0 0.5em 0.4em}.mw-parser-output .sidebar-content-with-subgroup{padding:0.1em 0.4em 0.2em}.mw-parser-output .sidebar-above,.mw-parser-output .sidebar-below{padding:0.3em 0.8em;font-weight:bold}.mw-parser-output .sidebar-collapse .sidebar-above,.mw-parser-output .sidebar-collapse .sidebar-below{border-top:1px solid #aaa;border-bottom:1px solid #aaa}.mw-parser-output .sidebar-navbar{text-align:right;font-size:115%;padding:0 0.4em 0.4em}.mw-parser-output .sidebar-list-title{padding:0 0.4em;text-align:left;font-weight:bold;line-height:1.6em;font-size:105%}.mw-parser-output .sidebar-list-title-c{padding:0 0.4em;text-align:center;margin:0 3.3em}@media(max-width:720px){body.mediawiki .mw-parser-output .sidebar{width:100%!important;clear:both;float:none!important;margin-left:0!important;margin-right:0!important}}</style><style data-mw-deduplicate="TemplateStyles:r1126788409">.mw-parser-output .plainlist ol,.mw-parser-output .plainlist ul{line-height:inherit;list-style:none;margin:0;padding:0}.mw-parser-output .plainlist ol li,.mw-parser-output .plainlist ul li{margin-bottom:0}</style><table class="sidebar sidebar-collapse nomobile nowraplinks" style="border:2px solid #90ee90"><tbody><tr><td class="sidebar-pretitle">Part of a series on</td></tr><tr><th class="sidebar-title-with-pretitle" style="background:#90ee90;padding:0.2em;font-size:175%;font-weight:bold">Evolutionary biology</th></tr><tr><td class="sidebar-image"><img alt="Darwin's finches by Gould.jpg" data-file-height="1319" data-file-width="1748" decoding="async" height="136" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Darwin%27s_finches_by_Gould.jpg/180px-Darwin%27s_finches_by_Gould.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Darwin%27s_finches_by_Gould.jpg/270px-Darwin%27s_finches_by_Gould.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Darwin%27s_finches_by_Gould.jpg/360px-Darwin%27s_finches_by_Gould.jpg 2x" width="180"/></td></tr><tr><td class="sidebar-above" style="display:block; margin-bottom:0.55em; background-color: transparent;">
</td></tr><tr><td class="sidebar-content" style="background:transparent;">
</td>
</tr><tr><td class="sidebar-content" style="background:transparent;">
</td>
</tr><tr><td class="sidebar-content" style="background:transparent;">
</td>
</tr><tr><td class="sidebar-content" style="background:transparent;">
</td>
</tr><tr><td class="sidebar-content" style="background:transparent;">
</td>
</tr><tr><td class="sidebar-below hlist">
<ul><li><span class="nowrap"><img alt="icon" class="noviewer" data-file-height="760" data-file-width="915" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/16px-Tree_of_life.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/24px-Tree_of_life.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/32px-Tree_of_life.svg.png 2x" width="16"/> Evolutionary biology portal</span></li>
<li><span class="nowrap"><img alt="" class="noviewer" data-file-height="185" data-file-width="180" decoding="async" height="16" src="//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/16px-Symbol_category_class.svg.png" srcset="//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/23px-Symbol_category_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/31px-Symbol_category_class.svg.png 2x" title="Category" width="16"/> Category</span></li></ul></td></tr><tr><td class="sidebar-navbar"><style data-mw-deduplicate="TemplateStyles:r1063604349">.mw-parser-output .navbar{display:inline;font-size:88%;font-weight:normal}.mw-parser-output .navbar-collapse{float:left;text-align:left}.mw-parser-output .navbar-boxtext{word-spacing:0}.mw-parser-output .navbar ul{display:inline-block;white-space:nowrap;line-height:inherit}.mw-parser-output .navbar-brackets::before{margin-right:-0.125em;content:"[ "}.mw-parser-output .navbar-brackets::after{margin-left:-0.125em;content:" ]"}.mw-parser-output .navbar li{word-spacing:-0.125em}.mw-parser-output .navbar a>span,.mw-parser-output .navbar a>abbr{text-decoration:inherit}.mw-parser-output .navbar-mini abbr{font-variant:small-caps;border-bottom:none;text-decoration:none;cursor:inherit}.mw-parser-output .navbar-ct-full{font-size:114%;margin:0 7em}.mw-parser-output .navbar-ct-mini{font-size:114%;margin:0 4em}</style></td></tr></tbody></table>


<p>A <b>phylogenetic tree</b> (also <b>phylogeny</b> or <b>evolutionary tree</b> <sup class="reference" id="cite_ref-Felsenstein_3-0">[3]</sup>) is a branching diagram or a tree showing the evolutionary relationships among various biological species or other entities based upon similarities and differences in their physical or genetic characteristics. All life on Earth is part of a single phylogenetic tree, indicating common ancestry.
</p><p>In a <i>rooted</i> phylogenetic tree, each node with descendants represents the inferred most recent common ancestor of those descendants,<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">[<i><span title="This claim needs references to reliable sources. (April 2021)">citation needed</span></i>]</sup> and the edge lengths in some trees may be interpreted as time estimates. Each node is called a taxonomic unit. Internal nodes are generally called hypothetical taxonomic units, as they cannot be directly observed. Trees are useful in fields of biology such as bioinformatics, systematics, and phylogenetics. <i>Unrooted</i> trees illustrate only the relatedness of the leaf nodes and do not require the ancestral root to be known or inferred.
</p>

<h2><span class="mw-headline" id="History">History</span><span class="mw-editsection"></span></h2>
<p>The idea of a "tree of life" arose from ancient notions of a ladder-like progression from lower into higher forms of life (such as in the Great Chain of Being). Early representations of "branching" phylogenetic trees include a "paleontological chart" showing the geological relationships among plants and animals in the book <i>Elementary Geology</i>, by Edward Hitchcock (first edition: 1840).
</p><p>Charles Darwin featured a diagrammatic evolutionary "tree" in his 1859 book <i>On the Origin of Species</i>. Over a century later, evolutionary biologists still use tree diagrams to depict evolution because such diagrams effectively convey the concept that speciation occurs through the adaptive and semirandom splitting of lineages. 
</p><p>The term <i>phylogenetic</i>, or <i>phylogeny</i>, derives from the two ancient greek words <span lang="grc">φῦλον</span> (<span lang="grc-Latn" title="Ancient Greek transliteration"><i>phûlon</i></span>), meaning "race, lineage", and <span lang="grc">γένεσις</span> (<span lang="grc-Latn" title="Ancient Greek transliteration"><i>génesis</i></span>), meaning "origin, source".<sup class="reference" id="cite_ref-4">[4]</sup><sup class="reference" id="cite_ref-5">[5]</sup>
</p>
<h2><span class="mw-headline" id="Properties">Properties</span><span class="mw-editsection"></span></h2>
<h3><span class="mw-headline" id="Rooted_tree">Rooted tree</span><span class="mw-editsection"></span></h3>

<p>A rooted phylogenetic tree (see two graphics at top) is a directed tree with a unique node — the root — corresponding to the (usually imputed) most recent common ancestor of all the entities at the leaves of the tree. The root node does not have a parent node, but serves as the parent of all other nodes in the tree. The root is therefore a node of degree 2, while other internal nodes have a minimum degree of 3 (where "degree" here refers to the total number of incoming and outgoing edges).
</p><p>The most common method for rooting trees is the use of an uncontroversial outgroup—close enough to allow inference from trait data or molecular sequencing, but far enough to be a clear outgroup. Another method is midpoint rooting, or a tree can also be rooted by using a non-stationary substitution model.<sup class="reference" id="cite_ref-6">[6]</sup>
</p>
<h3><span class="mw-headline" id="Unrooted_tree">Unrooted tree</span><span class="mw-editsection"></span></h3>

<p>Unrooted trees illustrate the relatedness of the leaf nodes without making assumptions about ancestry. They do not require the ancestral root to be known or inferred.<sup class="reference" id="cite_ref-8">[8]</sup> Unrooted trees can always be generated from rooted ones by simply omitting the root. By contrast, inferring the root of an unrooted tree requires some means of identifying ancestry. This is normally done by including an outgroup in the input data so that the root is necessarily between the outgroup and the rest of the taxa in the tree, or by introducing additional assumptions about the relative rates of evolution on each branch, such as an application of the molecular clock hypothesis.<sup class="reference" id="cite_ref-Maher_2002_9-0">[9]</sup>
</p>
<h3><span class="mw-headline" id="Bifurcating_versus_multifurcating">Bifurcating versus multifurcating</span><span class="mw-editsection"></span></h3>
<p>Both rooted and unrooted trees can be either bifurcating or multifurcating. A rooted bifurcating tree has exactly two descendants arising from each interior node (that is, it forms a binary tree), and an unrooted bifurcating tree takes the form of an unrooted binary tree, a free tree with exactly three neighbors at each internal node. In contrast, a rooted multifurcating tree may have more than two children at some nodes and an unrooted multifurcating tree may have more than three neighbors at some nodes.
</p>
<h3><span class="mw-headline" id="Labeled_versus_unlabeled">Labeled versus unlabeled</span><span class="mw-editsection"></span></h3>
<p>Both rooted and unrooted trees can be either labeled or unlabeled. A labeled tree has specific values assigned to its leaves, while an unlabeled tree, sometimes called a tree shape, defines a topology only. Some sequence-based trees built from a small genomic locus, such as Phylotree,<sup class="reference" id="cite_ref-10">[10]</sup> feature internal nodes labeled with inferred ancestral haplotypes.
</p>
<h3><span class="mw-headline" id="Enumerating_trees">Enumerating trees</span><span class="mw-editsection"></span></h3>

<p>The number of possible trees for a given number of leaf nodes depends on the specific type of tree, but there are always more labeled than unlabeled trees, more multifurcating than bifurcating trees, and more rooted than unrooted trees. The last distinction is the most biologically relevant; it arises because there are many places on an unrooted tree to put the root. For bifurcating labeled trees, the total number of rooted trees is:
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle (2n-3)!!={\frac {(2n-3)!}{2^{n-2}(n-2)!}}}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mo stretchy="false">(</mo>
<mn>2</mn>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>3</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
<mo>!</mo>
<mo>=</mo>
<mrow class="MJX-TeXAtom-ORD">
<mfrac>
<mrow>
<mo stretchy="false">(</mo>
<mn>2</mn>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>3</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
</mrow>
<mrow>
<msup>
<mn>2</mn>
<mrow class="MJX-TeXAtom-ORD">
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>2</mn>
</mrow>
</msup>
<mo stretchy="false">(</mo>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>2</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
</mrow>
</mfrac>
</mrow>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle (2n-3)!!={\frac {(2n-3)!}{2^{n-2}(n-2)!}}}</annotation>
</semantics>
</math></span><img alt="{\displaystyle (2n-3)!!={\frac {(2n-3)!}{2^{n-2}(n-2)!}}}" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/c6a987041030ccd0f7f93dea9650fc8e1b9872e2" style="vertical-align: -2.838ex; width:25.933ex; height:6.676ex;"/></span> for <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle n\geq 2}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mi>n</mi>
<mo>≥<!-- ≥ --></mo>
<mn>2</mn>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle n\geq 2}</annotation>
</semantics>
</math></span><img alt="n\geq 2" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/e6bf67f9d06ca3af619657f8d20ee1322da77174" style="vertical-align: -0.505ex; width:5.656ex; height:2.343ex;"/></span>, <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle n}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mi>n</mi>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle n}</annotation>
</semantics>
</math></span><img alt="n" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/a601995d55609f2d9f5e233e36fbe9ea26011b3b" style="vertical-align: -0.338ex; width:1.395ex; height:1.676ex;"/></span> represents the number of leaf nodes.<sup class="reference" id="cite_ref-Felsenstein1978_11-0">[11]</sup></dd></dl>
<p>For bifurcating labeled trees, the total number of unrooted trees is:<sup class="reference" id="cite_ref-Felsenstein1978_11-1">[11]</sup>
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle (2n-5)!!={\frac {(2n-5)!}{2^{n-3}(n-3)!}}}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mo stretchy="false">(</mo>
<mn>2</mn>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>5</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
<mo>!</mo>
<mo>=</mo>
<mrow class="MJX-TeXAtom-ORD">
<mfrac>
<mrow>
<mo stretchy="false">(</mo>
<mn>2</mn>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>5</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
</mrow>
<mrow>
<msup>
<mn>2</mn>
<mrow class="MJX-TeXAtom-ORD">
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>3</mn>
</mrow>
</msup>
<mo stretchy="false">(</mo>
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>3</mn>
<mo stretchy="false">)</mo>
<mo>!</mo>
</mrow>
</mfrac>
</mrow>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle (2n-5)!!={\frac {(2n-5)!}{2^{n-3}(n-3)!}}}</annotation>
</semantics>
</math></span><img alt="{\displaystyle (2n-5)!!={\frac {(2n-5)!}{2^{n-3}(n-3)!}}}" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/02a169c74c828e156fd7315c5bb939f93c485d26" style="vertical-align: -2.838ex; width:25.933ex; height:6.676ex;"/></span> for <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle n\geq 3}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mi>n</mi>
<mo>≥<!-- ≥ --></mo>
<mn>3</mn>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle n\geq 3}</annotation>
</semantics>
</math></span><img alt="n\geq 3" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/73136e4a27fe39c123d16a7808e76d3162ce42bb" style="vertical-align: -0.505ex; width:5.656ex; height:2.343ex;"/></span>.</dd></dl>
<p>Among labeled bifurcating trees, the number of unrooted trees with <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle n}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mi>n</mi>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle n}</annotation>
</semantics>
</math></span><img alt="n" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/a601995d55609f2d9f5e233e36fbe9ea26011b3b" style="vertical-align: -0.338ex; width:1.395ex; height:1.676ex;"/></span> leaves is equal to the number of rooted trees with <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle n-1}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mi>n</mi>
<mo>−<!-- − --></mo>
<mn>1</mn>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle n-1}</annotation>
</semantics>
</math></span><img alt="n-1" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/fbd0b0f32b28f51962943ee9ede4fb34198a2521" style="vertical-align: -0.505ex; width:5.398ex; height:2.343ex;"/></span> leaves.<sup class="reference" id="cite_ref-Felsenstein_3-1">[3]</sup>
</p><p>The number of rooted trees grows quickly as a function of the number of tips. For 10 tips, there are more than <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math alttext="{\displaystyle 34\times 10^{6}}" xmlns="http://www.w3.org/1998/Math/MathML">
<semantics>
<mrow class="MJX-TeXAtom-ORD">
<mstyle displaystyle="true" scriptlevel="0">
<mn>34</mn>
<mo>×<!-- × --></mo>
<msup>
<mn>10</mn>
<mrow class="MJX-TeXAtom-ORD">
<mn>6</mn>
</mrow>
</msup>
</mstyle>
</mrow>
<annotation encoding="application/x-tex">{\displaystyle 34\times 10^{6}}</annotation>
</semantics>
</math></span><img alt="{\displaystyle 34\times 10^{6}}" aria-hidden="true" class="mwe-math-fallback-image-inline" src="https://wikimedia.org/api/rest_v1/media/math/render/svg/9399400fca9cc37215c0952d138fe02ae49bf984" style="vertical-align: -0.338ex; width:8.544ex; height:2.676ex;"/></span> possible bifurcating trees, and the number of multifurcating trees rises faster, with ca. 7 times as many of the latter as of the former.
</p>
<style data-mw-deduplicate="TemplateStyles:r1105208659">.mw-parser-output .alternating>thead+tbody>tr:nth-child(even),.mw-parser-output .alternating>tbody:first-child>tr:nth-child(odd){background-color:#eee}</style>
<table class="wikitable sortable alternating" style="text-align: left; margin-left: auto; margin-right: auto; border: none;">
<caption>Counting trees.<sup class="reference" id="cite_ref-Felsenstein1978_11-2">[11]</sup>
</caption>
<tbody><tr>
<th>Labeled<br/>leaves</th>
<th>Binary<br/>unrooted trees</th>
<th>Binary<br/>rooted trees</th>
<th>Multifurcating<br/>rooted trees</th>
<th>All possible<br/>rooted trees
</th></tr>
<tr>
<td>1</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>1
</td></tr>
<tr>
<td>2</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>1
</td></tr>
<tr>
<td>3</td>
<td>1</td>
<td>3</td>
<td>1</td>
<td>4
</td></tr>
<tr>
<td>4</td>
<td>3</td>
<td>15</td>
<td>11</td>
<td>26
</td></tr>
<tr>
<td>5</td>
<td>15</td>
<td>105</td>
<td>131</td>
<td>236
</td></tr>
<tr>
<td>6</td>
<td>105</td>
<td>945</td>
<td>1,807</td>
<td>2,752
</td></tr>
<tr>
<td>7</td>
<td>945</td>
<td>10,395</td>
<td>28,813</td>
<td>39,208
</td></tr>
<tr>
<td>8</td>
<td>10,395</td>
<td>135,135</td>
<td>524,897</td>
<td>660,032
</td></tr>
<tr>
<td>9</td>
<td>135,135</td>
<td>2,027,025</td>
<td>10,791,887</td>
<td>12,818,912
</td></tr>
<tr>
<td>10</td>
<td>2,027,025</td>
<td>34,459,425</td>
<td>247,678,399</td>
<td>282,137,824
</td></tr>
</tbody></table>
<h2><span class="mw-headline" id="Special_tree_types">Special tree types</span><span class="mw-editsection"></span></h2>

<h3><span class="mw-headline" id="Dendrogram">Dendrogram</span><span class="mw-editsection"></span></h3>
<p>A dendrogram is a general name for a tree, whether phylogenetic or not, and hence also for the diagrammatic representation of a phylogenetic tree.<sup class="reference" id="cite_ref-12">[12]</sup>
</p>
<h3><span class="mw-headline" id="Cladogram">Cladogram</span><span class="mw-editsection"></span></h3>
<p>A cladogram only represents a branching pattern; i.e., its branch lengths do not represent time or relative amount of character change, and its internal nodes do not represent ancestors.<sup class="reference" id="cite_ref-13">[13]</sup>
</p>

<h3><span class="mw-headline" id="Phylogram">Phylogram</span><span class="mw-editsection"></span></h3>
<p>A phylogram is a phylogenetic tree that has branch lengths proportional to the amount of character change.<sup class="reference" id="cite_ref-15">[15]</sup>
</p><p>A chronogram is a phylogenetic tree that explicitly represents time through its branch lengths.<sup class="reference" id="cite_ref-Santamaria2009_16-0">[16]</sup>
</p>
<h3><span class="mw-headline" id="Dahlgrenogram">Dahlgrenogram</span><span class="mw-editsection"></span></h3>
<p>A Dahlgrenogram is a diagram representing a cross section of a phylogenetic tree.
</p>
<h3><span class="mw-headline" id="Phylogenetic_network">Phylogenetic network</span><span class="mw-editsection"></span></h3>
<p>A phylogenetic network is not strictly speaking a tree, but rather a more general graph, or a directed acyclic graph in the case of rooted networks. They are used to overcome some of the limitations inherent to trees.
</p>
<h3><span class="mw-headline" id="Spindle_diagram">Spindle diagram</span><span class="mw-editsection"></span></h3>

<p>A spindle diagram, or bubble diagram, is often called a romerogram, after its popularisation by the American palaeontologist Alfred Romer.<sup class="reference" id="cite_ref-palaeos_17-0">[17]</sup>
It represents taxonomic diversity (horizontal width) against geological time (vertical axis) in order to reflect the variation of abundance of various taxa through time.
However, a spindle diagram is not an evolutionary tree:<sup class="reference" id="cite_ref-3lbmonkeybrain_18-0">[18]</sup> the taxonomic spindles obscure the actual relationships of the parent taxon to the daughter taxon<sup class="reference" id="cite_ref-palaeos_17-1">[17]</sup> and have the disadvantage of involving the paraphyly of the parental group.<sup class="reference" id="cite_ref-Podani2019_19-0">[19]</sup>
This type of diagram is no longer used in the form originally proposed.<sup class="reference" id="cite_ref-Podani2019_19-1">[19]</sup>
</p>
<h3><span class="mw-headline" id="Coral_of_life">Coral of life</span><span class="mw-editsection"></span></h3>
<p> Darwin<sup class="reference" id="cite_ref-notebook_20-0">[20]</sup> also mentioned that the <i>coral</i> may be a more suitable metaphor than the <i>tree</i>. Indeed, phylogenetic corals are useful for portraying past and present life, and they have some advantages over trees (anastomoses allowed, etc.).<sup class="reference" id="cite_ref-Podani2019_19-2">[19]</sup>
</p><h2><span class="mw-headline" id="Construction">Construction</span><span class="mw-editsection"></span></h2>
<link href="mw-data:TemplateStyles:r1033289096" rel="mw-deduplicated-inline-style"/>
<p>Phylogenetic trees composed with a nontrivial number of input sequences are constructed using computational phylogenetics methods. Distance-matrix methods such as neighbor-joining or UPGMA, which calculate genetic distance from multiple sequence alignments, are simplest to implement, but do not invoke an evolutionary model. Many sequence alignment methods such as ClustalW also create trees by using the simpler algorithms (i.e. those based on distance) of tree construction. Maximum parsimony is another simple method of estimating phylogenetic trees, but implies an implicit model of evolution (i.e. parsimony). More advanced methods use the optimality criterion of maximum likelihood, often within a Bayesian framework, and apply an explicit model of evolution to phylogenetic tree estimation.<sup class="reference" id="cite_ref-Felsenstein_3-2">[3]</sup> Identifying the optimal tree using many of these techniques is NP-hard,<sup class="reference" id="cite_ref-Felsenstein_3-3">[3]</sup> so heuristic search and optimization methods are used in combination with tree-scoring functions to identify a reasonably good tree that fits the data.
</p><p>Tree-building methods can be assessed on the basis of several criteria:<sup class="reference" id="cite_ref-21">[21]</sup>
</p>
<ul><li>efficiency (how long does it take to compute the answer, how much memory does it need?)</li>
<li>power (does it make good use of the data, or is information being wasted?)</li>
<li>consistency (will it converge on the same answer repeatedly, if each time given different data for the same model problem?)</li>
<li>robustness (does it cope well with violations of the assumptions of the underlying model?)</li>
<li>falsifiability (does it alert us when it is not good to use, i.e. when assumptions are violated?)</li></ul>
<p>Tree-building techniques have also gained the attention of mathematicians. Trees can also be built using T-theory.<sup class="reference" id="cite_ref-22">[22]</sup>
</p>
<h3><span class="mw-headline" id="File_formats">File formats</span><span class="mw-editsection"></span></h3>
<style data-mw-deduplicate="TemplateStyles:r1097763485">.mw-parser-output .ambox{border:1px solid #a2a9b1;border-left:10px solid #36c;background-color:#fbfbfb;box-sizing:border-box}.mw-parser-output .ambox+link+.ambox,.mw-parser-output .ambox+link+style+.ambox,.mw-parser-output .ambox+link+link+.ambox,.mw-parser-output .ambox+.mw-empty-elt+link+.ambox,.mw-parser-output .ambox+.mw-empty-elt+link+style+.ambox,.mw-parser-output .ambox+.mw-empty-elt+link+link+.ambox{margin-top:-1px}html body.mediawiki .mw-parser-output .ambox.mbox-small-left{margin:4px 1em 4px 0;overflow:hidden;width:238px;border-collapse:collapse;font-size:88%;line-height:1.25em}.mw-parser-output .ambox-speedy{border-left:10px solid #b32424;background-color:#fee7e6}.mw-parser-output .ambox-delete{border-left:10px solid #b32424}.mw-parser-output .ambox-content{border-left:10px solid #f28500}.mw-parser-output .ambox-style{border-left:10px solid #fc3}.mw-parser-output .ambox-move{border-left:10px solid #9932cc}.mw-parser-output .ambox-protection{border-left:10px solid #a2a9b1}.mw-parser-output .ambox .mbox-text{border:none;padding:0.25em 0.5em;width:100%}.mw-parser-output .ambox .mbox-image{border:none;padding:2px 0 2px 0.5em;text-align:center}.mw-parser-output .ambox .mbox-imageright{border:none;padding:2px 0.5em 2px 0;text-align:center}.mw-parser-output .ambox .mbox-empty-cell{border:none;padding:0;width:1px}.mw-parser-output .ambox .mbox-image-div{width:52px}html.client-js body.skin-minerva .mw-parser-output .mbox-text-span{margin-left:23px!important}@media(min-width:720px){.mw-parser-output .ambox{margin:0 10%}}</style><table class="box-Unreferenced_section plainlinks metadata ambox ambox-content ambox-Unreferenced" role="presentation"><tbody><tr><td class="mbox-image"></td><td class="mbox-text"></td></tr></tbody></table>
<p>Trees can be encoded in a number of different formats, all of which must represent the nested structure of a tree. They may or may not encode branch lengths and other features. Standardized formats are critical for distributing and sharing trees without relying on graphics output that is hard to import into existing software. Commonly used formats are
</p>
<ul><li>Nexus file format</li>
<li>Newick format</li></ul>
<h2><span class="mw-headline" id="Limitations_of_phylogenetic_analysis">Limitations of phylogenetic analysis</span><span class="mw-editsection"></span></h2>
<link href="mw-data:TemplateStyles:r1097763485" rel="mw-deduplicated-inline-style"/><table class="box-More_citations_needed_section plainlinks metadata ambox ambox-content ambox-Refimprove" role="presentation"><tbody><tr><td class="mbox-image"></td><td class="mbox-text"></td></tr></tbody></table>
<p>Although phylogenetic trees produced on the basis of sequenced genes or genomic data in different species can provide evolutionary insight, these analyses have important limitations. Most importantly, the trees that they generate are not necessarily correct – they do not necessarily accurately represent the evolutionary history of the included taxa. As with any scientific result, they are subject to falsification by further study (e.g., gathering of additional data, analyzing the existing data with improved methods). The data on which they are based may be noisy;<sup class="reference" id="cite_ref-Townsend_2012_23-0">[23]</sup> the analysis can be confounded by genetic recombination,<sup class="reference" id="cite_ref-Arenas_2010_24-0">[24]</sup> horizontal gene transfer,<sup class="reference" id="cite_ref-Woese_2002_25-0">[25]</sup> hybridisation between species that were not nearest neighbors on the tree before hybridisation takes place, convergent evolution, and conserved sequences.
</p><p>Also, there are problems in basing an analysis on a single type of character, such as a single gene or protein or only on morphological analysis, because such trees constructed from another unrelated data source often differ from the first, and therefore great care is needed in inferring phylogenetic relationships among species. This is most true of genetic material that is subject to lateral gene transfer and recombination, where different haplotype blocks can have different histories. In these types of analysis, the output tree of a phylogenetic analysis of a single gene is an estimate of the gene's phylogeny (i.e. a gene tree) and not the phylogeny of the taxa (i.e. species tree) from which these characters were sampled, though ideally, both should be very close. For this reason, serious phylogenetic studies generally use a combination of genes that come from different genomic sources (e.g., from mitochondrial or plastid vs. nuclear genomes),<sup class="reference" id="cite_ref-26">[26]</sup> or genes that would be expected to evolve under different selective regimes, so that homoplasy (false homology) would be unlikely to result from natural selection.
</p><p>When extinct species are included as terminal nodes in an analysis (rather than, for example, to constrain internal nodes), they are considered not to represent direct ancestors of any extant species. Extinct species do not typically contain high-quality DNA.
</p><p>The range of useful DNA materials has expanded with advances in extraction and sequencing technologies. Development of technologies able to infer sequences from smaller fragments, or from spatial patterns of DNA degradation products, would further expand the range of DNA considered useful.
</p><p>Phylogenetic trees can also be inferred from a range of other data types, including morphology, the presence or absence of particular types of genes, insertion and deletion events – and any other observation thought to contain an evolutionary signal.
</p><p>Phylogenetic networks are used when bifurcating trees are not suitable, due to these complications which suggest a more reticulate evolutionary history of the organisms sampled.
</p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"></span></h2>
<style data-mw-deduplicate="TemplateStyles:r1128808480">.mw-parser-output .portalbox{padding:0;display:table;box-sizing:border-box;max-width:175px}.mw-parser-output .portalborder{border:solid #aaa 1px;padding:0.1em;background:#f9f9f9}.mw-parser-output .portalbox-entry{display:table-row;font-size:85%;line-height:110%;font-style:italic;font-weight:bold}.mw-parser-output .portalbox-image{display:table-cell;padding:0.2em;vertical-align:middle;text-align:center}.mw-parser-output .portalbox-link{display:table-cell;padding:0.2em 0.2em 0.2em 0.3em;vertical-align:middle}@media(min-width:720px){.mw-parser-output .portalleft{clear:left;float:left;margin:0.5em 1em 0.5em 0}.mw-parser-output .portalright{clear:right;float:right;margin:0.5em 0 0.5em 1em}}</style>
<style data-mw-deduplicate="TemplateStyles:r998391716">.mw-parser-output .div-col{margin-top:0.3em;column-width:30em}.mw-parser-output .div-col-small{font-size:90%}.mw-parser-output .div-col-rules{column-rule:1px solid #aaa}.mw-parser-output .div-col dl,.mw-parser-output .div-col ol,.mw-parser-output .div-col ul{margin-top:0}.mw-parser-output .div-col li,.mw-parser-output .div-col dd{page-break-inside:avoid;break-inside:avoid-column}</style>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"></span></h2>
<style data-mw-deduplicate="TemplateStyles:r1011085734">.mw-parser-output .reflist{font-size:90%;margin-bottom:0.5em;list-style-type:decimal}.mw-parser-output .reflist .references{font-size:100%;margin-bottom:0;list-style-type:inherit}.mw-parser-output .reflist-columns-2{column-width:30em}.mw-parser-output .reflist-columns-3{column-width:25em}.mw-parser-output .reflist-columns{margin-top:0.3em}.mw-parser-output .reflist-columns ol{margin-top:0}.mw-parser-output .reflist-columns li{page-break-inside:avoid;break-inside:avoid-column}.mw-parser-output .reflist-upper-alpha{list-style-type:upper-alpha}.mw-parser-output .reflist-upper-roman{list-style-type:upper-roman}.mw-parser-output .reflist-lower-alpha{list-style-type:lower-alpha}.mw-parser-output .reflist-lower-greek{list-style-type:lower-greek}.mw-parser-output .reflist-lower-roman{list-style-type:lower-roman}</style>
<h2><span class="mw-headline" id="Further_reading">Further reading</span><span class="mw-editsection"></span></h2>
<ul><li>Schuh, R. T. and A. V. Z. Brower. 2009. <i>Biological Systematics: principles and applications (2nd edn.)</i> <link href="mw-data:TemplateStyles:r1067248974" rel="mw-deduplicated-inline-style"/>ISBN 978-0-8014-4799-0</li>
<li>Manuel Lima, <i>The Book of Trees: Visualizing Branches of Knowledge</i>, 2014, Princeton Architectural Press, New York.</li>
<li>MEGA, a free software to draw phylogenetic trees.</li>
<li>Gontier, N. 2011. "Depicting the Tree of Life: the Philosophical and Historical Roots of Evolutionary Tree Diagrams."  Evolution, Education, Outreach 4: 515–538.</li></ul>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"></span></h2>
<style data-mw-deduplicate="TemplateStyles:r1097025294">.mw-parser-output .side-box{margin:4px 0;box-sizing:border-box;border:1px solid #aaa;font-size:88%;line-height:1.25em;background-color:#f9f9f9}.mw-parser-output .side-box-abovebelow,.mw-parser-output .side-box-text{padding:0.25em 0.9em}.mw-parser-output .side-box-image{padding:2px 0 2px 0.9em;text-align:center}.mw-parser-output .side-box-imageright{padding:2px 0.9em 2px 0;text-align:center}@media(min-width:500px){.mw-parser-output .side-box-flex{display:flex;align-items:center}.mw-parser-output .side-box-text{flex:1}}@media(min-width:720px){.mw-parser-output .side-box{width:238px}.mw-parser-output .side-box-right{clear:right;float:right;margin-left:1em}.mw-parser-output .side-box-left{margin-right:1em}}</style>
<h3><span class="mw-headline" id="Images">Images</span><span class="mw-editsection"></span></h3>
<ul><li>Human Y-Chromosome 2002 Phylogenetic Tree</li>
<li>iTOL: Interactive Tree Of Life</li>
<li>Phylogenetic Tree of Artificial Organisms Evolved on Computers Archived 2016-02-22 at the Wayback Machine</li>
<li>Miyamoto and Goodman's Phylogram of Eutherian Mammals</li></ul>
<h3><span class="mw-headline" id="General">General</span><span class="mw-editsection"></span></h3>
<ul><li>An overview of different methods of tree visualization is available at <link href="mw-data:TemplateStyles:r1067248974" rel="mw-deduplicated-inline-style"/><cite class="citation journal cs1" id="CITEREFPage2011">Page, R. D. M. (2011). "Space, time, form: Viewing the Tree of Life". <i>Trends in Ecology &amp; Evolution</i>. <b>27</b> (2): 113–120. doi:10.1016/j.tree.2011.12.002. PMID 22209094.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Trends+in+Ecology+%26+Evolution&amp;rft.atitle=Space%2C+time%2C+form%3A+Viewing+the+Tree+of+Life&amp;rft.volume=27&amp;rft.issue=2&amp;rft.pages=113-120&amp;rft.date=2011&amp;rft_id=info%3Adoi%2F10.1016%2Fj.tree.2011.12.002&amp;rft_id=info%3Apmid%2F22209094&amp;rft.aulast=Page&amp;rft.aufirst=R.+D.+M.&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3APhylogenetic+tree"></span></li>
<li>OneZoom: Tree of Life – all living species as intuitive and zoomable fractal explorer (responsive design)</li>
<li>Discover Life An interactive tree based on the U.S. National Science Foundation's Assembling the Tree of Life Project</li>
<li>PhyloCode</li>
<li>A Multiple Alignment of 139 Myosin Sequences and a Phylogenetic Tree</li>
<li>Tree of Life Web Project</li>
<li>Phylogenetic inferring on the T-REX server</li>
<li>NCBI's Taxonomy Database[1]</li>
<li>ETE: A Python Environment for Tree Exploration This is a programming library to analyze, manipulate and visualize phylogenetic trees. Ref.</li>
<li>A daily-updated tree of (sequenced) life <link href="mw-data:TemplateStyles:r1067248974" rel="mw-deduplicated-inline-style"/><cite class="citation journal cs1" id="CITEREFFangOatesPethicaGreenwood2013">Fang, H.; Oates, M. E.; Pethica, R. B.; Greenwood, J. M.; Sardar, A. J.; Rackham, O. J. L.; Donoghue, P. C. J.; Stamatakis, A.; De Lima Morais, D. A.; Gough, J. (2013). "A daily-updated tree of (sequenced) life as a reference for genome research". <i>Scientific Reports</i>. <b>3</b>: 2015. Bibcode:2013NatSR...3E2015F. doi:10.1038/srep02015. PMC <span class="cs1-lock-free" title="Freely accessible">6504836</span>. PMID 23778980.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Scientific+Reports&amp;rft.atitle=A+daily-updated+tree+of+%28sequenced%29+life+as+a+reference+for+genome+research&amp;rft.volume=3&amp;rft.pages=2015&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6504836%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F23778980&amp;rft_id=info%3Adoi%2F10.1038%2Fsrep02015&amp;rft_id=info%3Abibcode%2F2013NatSR...3E2015F&amp;rft.aulast=Fang&amp;rft.aufirst=H.&amp;rft.au=Oates%2C+M.+E.&amp;rft.au=Pethica%2C+R.+B.&amp;rft.au=Greenwood%2C+J.+M.&amp;rft.au=Sardar%2C+A.+J.&amp;rft.au=Rackham%2C+O.+J.+L.&amp;rft.au=Donoghue%2C+P.+C.+J.&amp;rft.au=Stamatakis%2C+A.&amp;rft.au=De+Lima+Morais%2C+D.+A.&amp;rft.au=Gough%2C+J.&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6504836&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3APhylogenetic+tree"></span></li></ul>


<style data-mw-deduplicate="TemplateStyles:r1110984671">.mw-parser-output .portal-bar{font-size:88%;font-weight:bold;display:flex;justify-content:center;align-items:baseline}.mw-parser-output .portal-bar-bordered{padding:0 2em;background-color:#fdfdfd;border:1px solid #a2a9b1;clear:both;margin:1em auto 0}.mw-parser-output .portal-bar-related{font-size:100%;justify-content:flex-start}.mw-parser-output .portal-bar-unbordered{padding:0 1.7em;margin-left:0}.mw-parser-output .portal-bar-header{margin:0 1em 0 0.5em;flex:0 0 auto;min-height:24px}.mw-parser-output .portal-bar-content{display:flex;flex-flow:row wrap;flex:0 1 auto;padding:0.15em 0;column-gap:1em;align-items:baseline}.mw-parser-output .portal-bar-content-related{}.mw-parser-output .portal-bar-item{display:inline-block;margin:0.15em 0.2em;min-height:24px;line-height:24px}@media screen and (max-width:768px){.mw-parser-output .portal-bar{font-size:88%;font-weight:bold;display:flex;flex-flow:column wrap;align-items:baseline}.mw-parser-output .portal-bar-header{text-align:center;flex:0;padding-left:0.5em;margin:0 auto}.mw-parser-output .portal-bar-related{font-size:100%;align-items:flex-start}.mw-parser-output .portal-bar-content{display:flex;flex-flow:row wrap;align-items:center;flex:0;column-gap:1em;border-top:1px solid #a2a9b1;margin:0 auto}.mw-parser-output .portal-bar-content-related{border-top:none;margin:0}}.mw-parser-output .navbox+link+.portal-bar,.mw-parser-output .navbox+style+.portal-bar,.mw-parser-output .navbox+link+.portal-bar-bordered,.mw-parser-output .navbox+style+.portal-bar-bordered,.mw-parser-output .sister-bar+link+.portal-bar,.mw-parser-output .sister-bar+style+.portal-bar,.mw-parser-output .portal-bar+.navbox-styles+.navbox,.mw-parser-output .portal-bar+.navbox-styles+.sister-bar{margin-top:-1px}</style>
<!-- 
NewPP limit report
Parsed by mw2377
Cached time: 20221223231613
Cache expiry: 1814400
Reduced expiry: false
Complications: [vary‐revision‐sha1, show‐toc]
CPU time usage: 0.629 seconds
Real time usage: 0.862 seconds
Preprocessor visited node count: 3199/1000000
Post‐expand include size: 129997/2097152 bytes
Template argument size: 6592/2097152 bytes
Highest expansion depth: 15/100
Expensive parser function count: 7/500
Unstrip recursion depth: 1/20
Unstrip post‐expand size: 109738/5000000 bytes
Lua time usage: 0.382/10.000 seconds
Lua memory usage: 17800132/52428800 bytes
Number of Wikibase entities loaded: 1/400
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00%  702.965      1 -total
 27.66%  194.453      1 Template:Reflist
 21.50%  151.165     18 Template:Cite_journal
 12.28%   86.321      2 Template:Wikt-lang
  9.29%   65.331      1 Template:Evolutionary_biology
  9.00%   63.264      1 Template:Sidebar_with_collapsible_lists
  7.66%   53.832      1 Template:PhylomapB
  6.99%   49.144      1 Template:Short_description
  5.97%   41.969      1 Template:Commons_category
  5.66%   39.791      1 Template:Sister_project
-->
<!-- Saved in parser cache with key enwiki:pcache:idhash:149326-0!canonical and timestamp 20221223231612 and revision id 1127018370.
 -->
</div></body>
</html>